Remotion/Web/Development.WebTesting/HostingStrategies/Configuration/TestSiteLayoutConfiguration.cs

```diff
@@ -41,6 +43,18 @@ public TestSiteLayoutConfiguration ([NotNull] IWebTestSettings webTestSettings)
       Resources = webTestSettings.TestSiteLayout.Resources
           .Select(resourceElement => EnsureRootedPath(RootPath, resourceElement))
           .Select(rootedPath => new TestSiteResource(rootedPath)).ToArray();
+
+      ProcessPath = GetRootedProcessPathOrNull(RootPath, webTestSettings.TestSiteLayout.ProcessPath);
+    }
+
+    private string? GetRootedProcessPathOrNull (string rootPath, string? processPath)
+    {
+      if (processPath == null)
+        return null;
+      if (!processPath.EndsWith(".exe"))
+        return null;
```

Comment:
I would suggest this is an exception, otherwise you'd get difficult to trace errors.